encryption_works.md

```diff
@@ -154,12 +154,10 @@ As well as protecting password vaults, you should also choose good passphrases f
 
 Another way to beef up the security of your accounts is to enable two-factor authentication, which changes your login process and provides you with a second layer of protection against people trying to change or brute-force your password. In addition to asking you for something you know (your password), you also have to provide something you have (physical access to your cellphone). This is checked by way of a one-time confirmation number that’s texted to you and which you have to enter during the login process.
 
-With 2FA, even if an attacker somehow learns your password, they won’t be able to log in without the cellphone associated with the account.
+With 2FA, even if an attacker somehow learns your password, they won’t be able to log in without the cellphone associated with the account. After the 2014 celebrity photo hack, Recode [reported](https://recode.net/2014/09/01/apple-says-it-is-actively-investigating-celeb-photo-hack/https://recode.net/2014/09/01/apple-says-it-is-actively-investigating-celeb-photo-hack/) that two-factor authentication would probably have prevented the attackers from gaining access to the iCloud accounts.
```

Comment:
double url! hazards of copy-pasting... (see ["go to fail"](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/))